modules/Tvpapi/resources/tvpapiAnalytics.js

```diff
@@ -103,9 +103,12 @@
 		},
 
 		sendMediaHit: function() {
-			// Do not send media hit in the following conditions: concurrent limit, current time not updated (might be error)
-			if(this.concurrentFlag || this.getPlayer().getPlayerElementTime() === 0 ){
-				return;
+		    var isLive = this.getPlayer().isLive();
+		    var isDvr = this.getPlayer().isDVR();
+
+			// Do not send media hit in the following conditions: concurrent limit, current time not updated (might be error) when media is not (live and not dvr)
+			if(this.concurrentFlag || !(isLive && !isDvr) && this.getPlayer().getPlayerElementTime() === 0) ){
```

Comment:
The ! negation should turn && to || so it is 
`!(isLive && !isDvr) ===> (!isLive || isDvr)`

if we already doing a complex expression simpler, let's move `currentTime = this.getPlayer().getPlayerElementTime()` and add it as `currentTime === 0` in the if expression.